include/swift/Basic/SupplementaryOutputPaths.h

```diff
@@ -0,0 +1,69 @@
+//===--- SupplementaryOutputPaths.h - Command to Execute -------*- C++ -*-===*\\
```

Comment:
What is "Command to Execute"? (Also, wrong slashes in banner.)

Reply:
For this PR, I removed the entire file. A later commit in here had deleted it, but now it's not there at all. For the PR in which this will reappear, I have fixed the header. Couldn't find any words that would fit in the required space, but I think the filename is clear enough.